### ISSUE TITLE
Dark cerumen 

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10160]
**Preferred term label:**

**Synonyms**
dark earwax, pigmented cerumen

**Definition (free text, please give PubMed ID)**
PMID: 20301627 : Ochronosis (bluish-black pigmentation of connective tissue). Accumulation of HGA and its oxidation 
products (e.g., benzoquinone acetic acid) in connective tissue leads to ochronosis (....)
(....)- Pigment also appears in **cerumen** and in perspiration, causing discoloration of clothing.

PMID: 37395296 Due to a mutation in the gene (HGD; Gene/Locus MIM number 607474) coding for homogentisate 1,2-dioxygenase (EC 1.13.11.5), there is a deficiency in this enzyme activity leading to a block of the catabolism in the phenylalanine and tyrosine
pathway. This results in increased homogentisic acid (HGA) generation followed by efficient excretion in the urine and its accumulation in the tissues of the human body. (....)
(...) **Dark earwax** was present in six (46%) patients and has usually been observed from infancy or early childhood

**Parent term (use hpo.jax.org/app)**
Abnormal cerumen color HP:0030790

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:56 Alkaptonuria

**Your nano-attribution (ORCID)**
https://orcid.org/0009-0005-6714-5727

**Comment by pnrobinson** [comment]
added as
HP:6000025

Dark cerumen
